Description: Adding some missing html dirs to the install script
--- a/html/Makefile.am
+++ b/html/Makefile.am
@@ -1,4 +1,4 @@
 htmldir=$(DESTDIR)/$(prefix)/share/boinc-server-maker/html
 install-data-local:
 	mkdir -p $(htmldir)
-	cp -r $(srcdir)/inc $(srcdir)/languages $(srcdir)/ops $(srcdir)/user $(htmldir)
+	cp -r $(srcdir)/drupal $(srcdir)/inc $(srcdir)/languages $(srcdir)/ops $(srcdir)/project.sample $(srcdir)/user $(htmldir)
